src/benchmark_metrics.py

```diff
@@ -78,6 +78,21 @@
     }
 }
 
+# Thresdhold for TRCOMP benchmarks
+# Metric: SM Billable secs
+# Unit: secs
+TRCOMP_THRESHOLD = {
+    'tensorflow': {                 # framework
+        '2.9': {                    # framework version
```

Comment:
Is this threshold specific to v2.9? or any version above v2.9?

Reply:
Yes 2.9.x will have this threshold. It will have to be recalibrated for every major release.